subsys/mgmt/osdp/src/osdp_phy.c

```diff
@@ -385,6 +385,12 @@ int osdp_phy_decode_packet(struct osdp_pd *pd, uint8_t *buf, int len)
 			pd->ephemeral_data[0] = OSDP_PD_NAK_SC_COND;
 			return OSDP_ERR_PKT_FMT;
 		}
+		if (!sc_is_active(pd) && pkt->data[1] > SCS_14) {
+			LOG_ERR("Received invalid secure message!");
```

Comment:
For CMD_SCRYPT there is LOG_WRN and here is LOG_ERR.
Both are cases where command is NACKed.

Reply:
Should have been an error level log too. Fixed.